tests/e2e/conftest.py

```
@@ -79,7 +80,12 @@ def _cluster_ols_install_or_adapt() -> tuple[str, str, str, str | None]:
     creds_list = creds.split()
     for i, prov in enumerate(provider_list):
         ols_installer.create_secrets(prov, creds_list[i], len(provider_list))
-    ols_url, token, metrics_token = adapt_ols_config()
+    try:
+        ols_url, token, metrics_token = adapt_ols_config()
+    except Exception as first_err:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)